src/components/settings/Settings.h

```diff
@@ -283,10 +294,21 @@ namespace Pinetime {
         return bleRadioEnabled;
       };
 
+      HeartRateBackgroundMeasurementInterval GetHeartRateBackgroundMeasurementInterval() const {
+        return settings.heartRateBackgroundMeasurementInterval;
+      }
+
+      void SetHeartRateBackgroundMeasurementInterval(HeartRateBackgroundMeasurementInterval newHeartRateBackgroundMeasurementInterval) {
+        if (newHeartRateBackgroundMeasurementInterval != settings.heartRateBackgroundMeasurementInterval) {
+          settingsChanged = true;
+        }
+        settings.heartRateBackgroundMeasurementInterval = newHeartRateBackgroundMeasurementInterval;
+      }
+
     private:
       Pinetime::Controllers::FS& fs;
 
-      static constexpr uint32_t settingsVersion = 0x0007;
+      static constexpr uint32_t settingsVersion = 0x0008;
```

Comment:
Does this need to be bumped to `0x0009`? Looks like `0x0008` is in `main` already:
https://github.com/InfiniTimeOrg/InfiniTime/blob/main/src/components/settings/Settings.h#L304

Reply:
Yes you are right. Thank you!
I will keep this suggested change open until the PR is otherwise ready to merge.

I bumped the settings version a few times already and I don't want to do it every time there is the settings file in main anymore.